--- mingw-runtime-3.0/mingwex/configure.in.wiget	2003-05-06 15:46:46.000000000 +0000
+++ mingw-runtime-3.0/mingwex/configure.in	2003-07-04 13:47:53.000000000 +0000
@@ -19,21 +19,25 @@
 AC_PREREQ(2.13)
 AC_INIT(imaxabs.c)
 
-CC=${CC-cc}
+AC_CANONICAL_SYSTEM
+AC_CHECK_TOOL(CC,gcc,gcc)
+if test -z "$CC"; then
+	AC_CHECK_TOOL(CC,cc,cc)
+fi
 AC_SUBST(CC)
-AS=${AS-as}
+AC_CHECK_TOOL(AS,as,as)
 AC_SUBST(AS)
-AR=${AR-ar}
+AC_CHECK_TOOL(AR,ar,ar)
 AC_SUBST(AR)
-LD=${LD-ld}
+AC_CHECK_TOOL(LD,ld,ld)
 AC_SUBST(LD)
-RANLIB=${RANLIB-ranlib}
+AC_CHECK_TOOL(RANLIB,ranlib,ranlib)
 AC_SUBST(RANLIB)
-DLLTOOL=${DLLTOOL-dlltool}
+AC_CHECK_TOOL(DLLTOOL,dlltool,dlltool)
 AC_SUBST(DLLTOOL)
-DLLWRAP=${DLLWRAP-dllwrap}
+AC_CHECK_TOOL(DLLWRAP,dllwrap,dllwrap)
 AC_SUBST(DLLWRAP)
-AC_CANONICAL_SYSTEM
+AC_SUBST(CFLAGS)
 
 case "$target_os" in
 *cygwin*)
--- mingw-runtime-3.0/profile/configure.in.wiget	2003-05-06 15:46:46.000000000 +0000
+++ mingw-runtime-3.0/profile/configure.in	2003-07-04 13:46:49.000000000 +0000
@@ -19,6 +19,24 @@
 AC_PREREQ(2.13)
 AC_INIT(gcrt0.c)
 
+AC_CANONICAL_SYSTEM
+AC_CHECK_TOOL(CC,gcc,gcc)
+if test -z "$CC"; then
+	AC_CHECK_TOOL(CC,cc,cc)
+fi
+AC_SUBST(CC)
+AC_CHECK_TOOL(AS,as,as)
+AC_SUBST(AS)
+AC_CHECK_TOOL(AR,ar,ar)
+AC_SUBST(AR)
+AC_CHECK_TOOL(LD,ld,ld)
+AC_SUBST(LD)
+AC_CHECK_TOOL(RANLIB,ranlib,ranlib)
+AC_SUBST(RANLIB)
+AC_CHECK_TOOL(DLLTOOL,dlltool,dlltool)
+AC_SUBST(DLLTOOL)
+AC_CHECK_TOOL(DLLWRAP,dllwrap,dllwrap)
+AC_SUBST(DLLWRAP)
 CC=${CC-cc}
 AC_SUBST(CC)
 AS=${AS-as}
--- mingw-runtime-3.0/configure.in.wiget	2003-05-06 15:23:18.000000000 +0000
+++ mingw-runtime-3.0/configure.in	2003-07-04 13:46:49.000000000 +0000
@@ -26,15 +26,13 @@
 dnl itself.
 
 AC_DEFUN(LIB_AC_PROG_CC,
-[AC_BEFORE([$0], [AC_PROG_CPP])dnl
+[
 AC_CHECK_TOOL(CC, gcc, gcc)
 if test -z "$CC"; then
   AC_CHECK_PROG(CC, cc, cc, , , /usr/ucb/cc)
   test -z "$CC" && AC_MSG_ERROR([no acceptable cc found in \$PATH])
 fi
 
-AC_PROG_CC_GNU
-
 if test $ac_cv_prog_gcc = yes; then
   GCC=yes
 dnl Check whether -g works, even if CFLAGS is set, in case the package
@@ -43,7 +41,6 @@
   ac_test_CFLAGS="${CFLAGS+set}"
   ac_save_CFLAGS="$CFLAGS"
   CFLAGS=
-  AC_PROG_CC_G
   if test "$ac_test_CFLAGS" = set; then
     CFLAGS="$ac_save_CFLAGS"
   elif test $ac_cv_prog_cc_g = yes; then
@@ -69,8 +66,8 @@
 AC_SUBST(all_dlls_host)
 AC_SUBST(install_dlls_host)
 
-AC_CHECK_TOOL(AR, ar, ar)
 
+AC_CHECK_TOOL(AR, ar, ar)
 AC_SUBST(AR)
 AC_CHECK_TOOL(AS, as, as)
 AC_SUBST(AS)
@@ -84,10 +81,11 @@
 AC_SUBST(DLLWRAP)
 AC_CHECK_TOOL(WINDRES, windres, windres)
 AC_SUBST(WINDRES)
+AC_CANONICAL_SYSTEM
+AC_SUBST(CFLAGS)
 
-AC_ALLOCA
+dnl AC_ALLOCA
 
-AC_CANONICAL_SYSTEM
 SUBDIRS="profile mingwex"
 configdirs="profile mingwex"
 HEADER_SUBDIR=""
