--- w32api-4.0.3-1.mingw32-src/misc/src/libcrt/gdtoa/arithchk.c.orig	2012-10-07 20:55:06.000000000 +0200
+++ w32api-4.0.3-1.mingw32-src/misc/src/libcrt/gdtoa/arithchk.c	2014-01-30 17:48:48.932622127 +0100
@@ -106,7 +106,7 @@
 	long Cray1;
 
 	/* Cray1 = 4617762693716115456 -- without overflow on non-Crays */
-	Cray1 = printf(emptyfmt) < 0 ? 0 : 4617762;
+	Cray1 = printf(emptyfmt, "") < 0 ? 0 : 4617762;
 	if (printf(emptyfmt, Cray1) >= 0)
 		Cray1 = 1000000*Cray1 + 693716;
 	if (printf(emptyfmt, Cray1) >= 0)
