--- mingw-runtime-3.0/mingwex/configure.in.wiget	2003-05-06 15:46:46.000000000 +0000
+++ mingw-runtime-3.0/mingwex/configure.in	2003-07-04 13:47:53.000000000 +0000
@@ -19,21 +19,25 @@
 AC_PREREQ(2.13)
 AC_INIT(imaxabs.c)
 
-CC=${CC-cc}
+AC_CANONICAL_SYSTEM
+AC_CHECK_TOOL(CC,gcc,gcc)
+if test -z "$CC"; then
+	AC_CHECK_TOOL(CC,cc,cc)
+fi
 AC_SUBST(CC)
-AS=${AS-as}
+AC_CHECK_TOOL(AS,as,as)
 AC_SUBST(AS)
-AR=${AR-ar}
+AC_CHECK_TOOL(AR,ar,ar)
 AC_SUBST(AR)
-LD=${LD-ld}
+AC_CHECK_TOOL(LD,ld,ld)
 AC_SUBST(LD)
-RANLIB=${RANLIB-ranlib}
+AC_CHECK_TOOL(RANLIB,ranlib,ranlib)
 AC_SUBST(RANLIB)
-DLLTOOL=${DLLTOOL-dlltool}
+AC_CHECK_TOOL(DLLTOOL,dlltool,dlltool)
 AC_SUBST(DLLTOOL)
-DLLWRAP=${DLLWRAP-dllwrap}
+AC_CHECK_TOOL(DLLWRAP,dllwrap,dllwrap)
 AC_SUBST(DLLWRAP)
-AC_CANONICAL_SYSTEM
+AC_SUBST(CFLAGS)
 
 case "$target_os" in
 *cygwin*)
--- mingw-runtime-3.0/profile/configure.in.wiget	2003-05-06 15:46:46.000000000 +0000
+++ mingw-runtime-3.0/profile/configure.in	2003-07-04 13:46:49.000000000 +0000
@@ -19,6 +19,24 @@
 AC_PREREQ(2.13)
 AC_INIT(gcrt0.c)
 
+AC_CANONICAL_SYSTEM
+AC_CHECK_TOOL(CC,gcc,gcc)
+if test -z "$CC"; then
+	AC_CHECK_TOOL(CC,cc,cc)
+fi
+AC_SUBST(CC)
+AC_CHECK_TOOL(AS,as,as)
+AC_SUBST(AS)
+AC_CHECK_TOOL(AR,ar,ar)
+AC_SUBST(AR)
+AC_CHECK_TOOL(LD,ld,ld)
+AC_SUBST(LD)
+AC_CHECK_TOOL(RANLIB,ranlib,ranlib)
+AC_SUBST(RANLIB)
+AC_CHECK_TOOL(DLLTOOL,dlltool,dlltool)
+AC_SUBST(DLLTOOL)
+AC_CHECK_TOOL(DLLWRAP,dllwrap,dllwrap)
+AC_SUBST(DLLWRAP)
 CC=${CC-cc}
 AC_SUBST(CC)
 AS=${AS-as}
--- mingw-runtime-3.0/configure.in.wiget	2003-05-06 15:23:18.000000000 +0000
+++ mingw-runtime-3.0/configure.in	2003-07-04 13:46:49.000000000 +0000
@@ -19,8 +19,6 @@
 AC_PREREQ(2.59)
 AC_INIT(dllmain.c)
 
-LIB_AC_PROG_CC
-
 case "$with_cross_host" in
   ""|*cygwin*) all_dlls_host='all_dlls_host'
 	       install_dlls_host='install_dlls_host';;
@@ -28,6 +26,8 @@
 AC_SUBST(all_dlls_host)
 AC_SUBST(install_dlls_host)
 
+AC_CHECK_TOOL(CC, gcc, gcc)
+AC_SUBST(CC)
 AC_CHECK_TOOL(AR, ar, ar)
 
 AC_SUBST(AR)
@@ -43,10 +43,10 @@
 AC_SUBST(DLLWRAP)
 AC_CHECK_TOOL(WINDRES, windres, windres)
 AC_SUBST(WINDRES)
+AC_CANONICAL_SYSTEM
+AC_SUBST(CFLAGS)
 
-AC_ALLOCA
 
-AC_CANONICAL_SYSTEM
 AC_CONFIG_SUBDIRS(profile mingwex)
 HEADER_SUBDIR=""
 
