--- mingwrt-3.16-mingw32/Makefile.in.org	2009-08-23 02:15:25.799505624 +0200
+++ mingwrt-3.16-mingw32/Makefile.in	2009-08-23 02:15:37.278392489 +0200
@@ -166,7 +166,6 @@
 W32API_INCLUDE = @W32API_INCLUDE@
 INCLUDES = -I$(srcdir)/include \
   -I$(srcdir)/../include \
-  -nostdinc \
   -iwithprefixbefore include
 ALL_CFLAGS = $(CFLAGS) $(INCLUDES) $(W32API_INCLUDE)
 ALL_CXXFLAGS = $(CXXFLAGS) $(INCLUDES) -nostdinc++ $(W32API_INCLUDE)
