--- include/sys/types.h.orig	2017-07-09 22:38:20.000000000 +0200
+++ include/sys/types.h	2018-09-14 22:18:03.620329409 +0200
@@ -222,5 +222,10 @@
 #endif
 #endif  /* _SYS_TYPES_H normal inclusion */
 
+#ifndef _ERRCODE_DEFINED
+#define _ERRCODE_DEFINED
+typedef int errno_t;
+#endif
+
 #endif	/* ! RC_INVOKED */
 #endif	/* !_SYS_TYPES_H: $RCSfile: types.h,v $: end of file */
--- include/stdlib.h.orig	2018-09-14 22:19:47.526994890 +0200
+++ include/stdlib.h	2018-09-14 22:20:54.966994122 +0200
@@ -481,6 +481,10 @@
 _CRTIMP __cdecl __MINGW_NOTHROW  int rand (void);
 _CRTIMP __cdecl __MINGW_NOTHROW  void srand (unsigned int);
 
+#if defined (_CRT_RAND_S) && (WINVER >= 0x0501)
+_CRTIMP __cdecl __MINGW_NOTHROW  errno_t rand_s (unsigned int*);
+#endif
+
 _CRTIMP __cdecl __MINGW_NOTHROW  void *calloc (size_t, size_t) __MINGW_ATTRIB_MALLOC;
 _CRTIMP __cdecl __MINGW_NOTHROW  void *malloc (size_t) __MINGW_ATTRIB_MALLOC;
 _CRTIMP __cdecl __MINGW_NOTHROW  void *realloc (void *, size_t);
